assertion/function/analyzer.go

```diff
@@ -137,9 +137,9 @@ func run(pass *analysis.Pass) (result interface{}, _ error) {
 		// Construct config for analyzing the functions in this file. By default, enable all checks
 		// on NilAway itself.
 		functionConfig := assertiontree.FunctionConfig{}
-		if strings.HasPrefix(pass.Pkg.Path(), config.NilAwayPkgPathPrefix) {
-			functionConfig.StructInitCheckType = config.DepthOneFieldCheck
-			// TODO: enable anonymous function flag here when  is done
+		if strings.HasPrefix(pass.Pkg.Path(), config.NilAwayPkgPathPrefix) { //nolint:revive
+			// TODO: enable struct initialization flag.
```

Comment:
We had to disable struct init for NilAway itself due to too many false postives (especially related to escaped fields). We should investigate this in the future and enable it.

Reply:
Do we have an issue tracking this in GH? If so, let's add the issue number on the TODO comment.